webcrawler.py

```diff
@@ -51,8 +58,9 @@ def collate():
         results = json.load(fd)
     for result in results:
         url = result['Link']
-        result_keywords[url].add(result['word'])
-        result_titles[url] = result['title']
+        if (contains_date(url)):
```

Comment:
Nitpick: the convention in Python is not to have parenthesis around `if` conditions, unless absolutely necessary.